core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/UtilV4Controller.java

```diff
@@ -129,4 +135,9 @@ public Response renewCertificate(RenewCertificateV4Request renewCertificateV4Req
         return Response.ok().build();
     }
 
+    @Override
+    @DisableCheckPermissions
```

Comment:
I am discussing it with @lnardai on how to implement authorization, and he will review it
It was only opened as a draft PR so that @pkedvessy can help with some issues I have at the moment